lametro/templates/search/_search_filter.html

```diff
@@ -8,14 +8,16 @@ <h3 class="accordion-header">
             <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#collapse_{{facet_name}}" aria-expanded="false" aria-controls="collapse_{{facet_name}}">
                 Filter by
                 {% if 'Legislation' in facet_label %}
-                    Report Type
+                    Board Report Type
```

Comment:
Making this facet read exactly the same as the corresponding topic category on the bill detail page would make it clear to Metro that the same tooltip would apply to both spots.